flowetl/etl/tests/test_construct_etl_dag.py

```diff
@@ -55,8 +71,20 @@ def test_construct_etl_dag_with_production_callables():
 
     assert dag.dag_id == f"etl_{cdr_type}"
 
-    dag_task_callable_mapping = {t.task_id: t.python_callable for t in dag.tasks}
-    assert dag_task_callable_mapping == PRODUCTION_ETL_TASK_CALLABLES
+    dag_task_callable_mapping = {
+        t.task_id: t.python_callable
+        for t in dag.tasks
+        if not isinstance(t, PostgresOperator)
```

Comment:
See above:
```suggestion
        if not instance(t, PythonOperator)
```

And similarly in L84 below. (Need to add `from airflow.operators.python_operator import PythonOperator` in the import section, too.)